software/desktop-environments/en.md

```diff
@@ -30,4 +30,26 @@ Solus offers the MATE desktop environment / experience, tailored to advanced use
 
 ``` bash
 sudo eopkg install -c desktop.mate
-```
\ No newline at end of file
+```
+
+## Plasma
+
+Solus will offer the KDE Plasma desktop environment / experience in the next comming release. Plasma is a powerful and highly customizable desktop enviroment built with Qt and the KDE framework. You can try it right now by running the following command:
+
+``` bash
+sudo eopkg install -c desktop.kde
```

Comment:
Doubtful this is the only component you need to install. Should bring in @sunnyflunk.

Reply:
This is wrong, should be desktop.kde.core and the recommended method will always be to use the ISO as it differs considerably from the other ISO's